src/block/tabs/style.js

```diff
@@ -43,6 +43,18 @@ const Styles = props => {
 				format="%spx"
 				responsive="all"
 			/>
+			<BlockCss
+				{ ...propsToPass }
+				selector=".%s .stk-block-tab-content > .stk-block-content > .stk-block-column[hidden]"
```

Comment:
We cannot have `>` signs in the selector for the **save** render. WP has a bug that it will throw an error in multisite.

Please change the implementation so we do not have `>` in the selector. Note that this is okay for **edit**